app/services/category_hashtag_data_source.rb

```diff
@@ -39,12 +39,16 @@ def self.lookup(guardian, slugs)
       .map { |category| category_to_hashtag_item(guardian_categories, category) }
   end
 
-  def self.search(guardian, term, limit)
+  def self.search(guardian, term, limit, exact_match: false)
     guardian_categories = Site.new(guardian).categories
 
     guardian_categories
       .select do |category|
-        category[:name].downcase.include?(term) || category[:slug].downcase.include?(term)
+        if exact_match
+          category[:slug].downcase == term
```

Comment:
Shouldn't we downcase term here too ?

Im a little bit confused on the name of the options here `exact_match` where the consequence is matching term on downcased slug, and the other branch is everything downcase and on name too. I wouldn't have guessed this behavior from `exact_match` name

Reply:
So maybe it should be: `exact_slug_match`

Reply:
`term` is already downcased by `HashtagAutocompleteService` at this point.

> So maybe it should be: exact_slug_match

This makes sense, I will change it to this for here and the other data sources. The intent is to find single results that exactly match the slug. Maybe I am going about this wrong, I had thought of using `.lookup` for this yesterday but then went away from it for...some reason. Will re-examine, maybe we won't need this extra param then.